cli/cmd/get.go

```diff
@@ -480,14 +489,73 @@ func describeAPI(name string, resourcesRes *schema.GetResourcesResponse) (string
 		sort.Strings(samplePlaceholderFields)
 		samplesPlaceholderStr := `{ "samples": [ { ` + strings.Join(samplePlaceholderFields, ", ") + " } ] }"
 		out += "Payload:  " + samplesPlaceholderStr + "\n"
-	}
-	if api != nil {
-		out += titleStr("Configuration") + api.UserConfigStr()
+	} else {
+		out += titleStr("Model Input Signature")
+
+		if groupStatus.Available() == 0 {
+			out += "Model "
+			return out, nil
+		}
+
+		modelInput, err := getModelInput(urls.Join(apiEndpoint, "signature"))
+		if err != nil {
+			return "", errors.Wrap(err, "")
+		}
+
+		rows := make([][]interface{}, len(modelInput.Signature))
+		rowNum := 0
+		for inputName, featureSignature := range modelInput.Signature {
+			shapeStr := make([]string, len(featureSignature.Shape))
+			for idx, dim := range featureSignature.Shape {
+				if dim == 0 {
+					shapeStr[idx] = "?"
+				} else {
+					shapeStr[idx] = fmt.Sprintf("%d", dim)
+				}
+			}
+			rows[rowNum] = []interface{}{
+				inputName,
+				featureSignature.Type,
+				s.ObjFlatNoQuotes(shapeStr),
+			}
+			rowNum++
+		}
+
+		t := table.Table{
+			Headers: []table.Header{
+				{Title: "FEATURE", MaxWidth: 32},
+				{Title: "TYPE", MaxWidth: 10},
+				{Title: "SHAPE", MaxWidth: 20},
+			},
+			Rows: rows,
+		}
+
+		out += table.MustFormat(t)
```

Comment:
It it worth also showing what the curl would look like, like we do for e2e? Since you printed the table we wouldn't need to include the shape information in the sample curl.

Reply:
The sample curl I am imagining for say xgboost iris would be something like this:

```
{
  input: [int]
}
```

That doesn't look like new information that isn't already described in the table. Additionally, it can not just be copied and pasted. We can discuss this more.